core/comment.js

```diff
@@ -251,44 +309,60 @@ Blockly.Comment.prototype.getBubbleSize = function() {
  * @param {number} height Height of the bubble.
  */
 Blockly.Comment.prototype.setBubbleSize = function(width, height) {
-  if (this.textarea_) {
+  if (this.bubble_) {
     this.bubble_.setBubbleSize(width, height);
   } else {
-    this.width_ = width;
-    this.height_ = height;
+    this.model_.size.width = width;
+    this.model_.size.height = height;
   }
 };
 
 /**
  * Returns this comment's text.
  * @return {string} Comment text.
+ * @deprecated August 2019 Use block.getCommentText() instead.
  */
 Blockly.Comment.prototype.getText = function() {
-  return this.textarea_ ? this.textarea_.value : this.text_;
+  return this.model_.text;
```

Comment:
``this.model_.text`` can be null, ``return this.model_.text || ''``

Reply:
That shouldn't happen in this context because a `null` value signals that there is no comment (and by extension no comment icon) but I'll add an `||` for safety.